apps/web/src/pages/templates/components/VariantItemCard.tsx

```diff
@@ -131,7 +131,11 @@ export const VariantItemCard = ({
     channel: StepTypeEnum;
     stepUuid: string;
   }>();
-  const subtitle = useStepSubtitle(variant, channel);
+  const subtitle = useStepSubtitle({
+    path: `steps.${stepIndex}.variants.${variantIndex}`,
+    step: variant,
+    channelType: channel,
+  });
```

Comment:
This hook is used in the step node and the variant node. It is responsible for creating the node content that's why I'm passing here the form path so it can calculate that text.

![Screenshot 2023-10-30 at 22 55 02](https://github.com/novuhq/novu/assets/2607232/6d54d611-caf9-4b88-bb61-f2df1de656ae)
![Screenshot 2023-10-30 at 22 54 45](https://github.com/novuhq/novu/assets/2607232/dec0013b-a417-4afa-a50d-c3933e18add6)